frontend/src/components/AdminApplicationOverview/reducer.ts

```diff
@@ -37,13 +41,13 @@ export const ApplicationReducer = (
         ...state,
         applications,
       };
-    case 'SET_APPLICATION_DATA': {
+    case SET_APPLICATION_DATA: {
       return {
         ...state,
         registeredApplication: action.payload,
       };
     }
-    case 'REMOVE_APPLICATION_DATA': {
+    case REMOVE_APPLICATION_DATA: {
```

Comment:
This does technically not belong in this PR.

Reply:
Technically not, but it shouldn't have been like that from the beginning